torch/nn/modules/transformer.py

```diff
@@ -313,16 +331,22 @@ def __init__(self, d_model: int, nhead: int, dim_feedforward: int = 2048, dropou
 
         # Legacy string support for activation function.
         if isinstance(activation, str):
-            self.activation = _get_activation_fn(activation)
-        else:
-            self.activation = activation
+            activation = _get_activation_fn(activation)
 
+        if activation is F.relu:
+            self.activation_relu_or_gelu = 1
+        elif activation is F.gelu:
+            self.activation_relu_or_gelu = 2
+        else:
+            self.activation_relu_or_gelu = 0
```

Comment:
prolly worth a little comment saying that this is to appease TorchScript

Reply:
done!